kratos/includes/table_stream.h

```diff
@@ -84,6 +85,9 @@ class TableStream
             mSeparator(Separator),
             mBoldFont(UseBoldFont)
     {
+        // Initialize STD output
+        ColorUtilities::InitializeSTDOutput();
```

Comment:
overall I am fine with this PR but this function confuses me, esp how / when it should be used

Reply:
@roigcarlo could you please also take a look?

Reply:
Basically always youy need bold/colors, In Unix does nothing, in Windows prepares the terminal for ANSI output